src/deploy-commands.js

```diff
@@ -1,34 +1,61 @@
 const { REST, Routes } = require('discord.js');
-const fs = require('node:fs');
-const path = require('node:path');
 const { DISCORD_SERVER } = require('./config');
 
+const pingCommand = require('./commands/utility/ping');
+const assingMeTaskCommand = require('./commands/utility/assignMeTaskThread');
+const changeStatusCommand = require('./commands/utility/changeStatus');
+const convertTimeCommand = require('./commands/utility/convert-time');
+const myPointsCommand = require('./commands/utility/my-points');
+const prTemplateCommand = require('./commands/utility/pr-template');
+const remindMeCommand = require('./commands/utility/remindme');
+const requestPointCommand = require('./commands/utility/requestPoint');
+const searchCodeCommand = require('./commands/utility/search-code-review');
+const searchMyPointsCommand = require('./commands/utility/search-my-points');
+const sendMessageCommand = require('./commands/utility/send-message');
+const serverCommand = require('./commands/utility/server');
+const userCommand = require('./commands/utility/user');
+const votePointsCommand = require('./commands/utility/vote-points');
+
 module.exports = (client) => {
   const commands = [];
-  const foldersPath = path.join(__dirname, 'commands');
-  const commandFolders = fs.readdirSync(foldersPath);
-  const token = DISCORD_SERVER.discordToken;
-  const clientId = DISCORD_SERVER.discordClientId;
-  const guildId = DISCORD_SERVER.discordGuildId;
-
-  for (const folder of commandFolders) {
-    const commandsPath = path.join(foldersPath, folder);
-    const commandFiles = fs
-      .readdirSync(commandsPath)
-      .filter((file) => file.endsWith('.js'));
-    for (const file of commandFiles) {
-      const command = require(path.join(commandsPath, file));
-      if ('data' in command && 'execute' in command) {
-        commands.push(command.data.toJSON());
-        client.commands.set(command.data.name, command);
-      } else {
-        console.log(
-          `[WARNING] The command at ${filePath} is missing a required "data" or "execute" property.`
-        );
-      }
+
+  const commandList = [
+    pingCommand,
+    assingMeTaskCommand,
+    changeStatusCommand,
+    convertTimeCommand,
+    myPointsCommand,
+    prTemplateCommand,
+    remindMeCommand,
+    requestPointCommand,
+    searchCodeCommand,
+    searchMyPointsCommand,
+    sendMessageCommand,
+    serverCommand,
+    userCommand,
+    votePointsCommand,
+  ]; // Agrega más comandos aquí
+
+  for (const command of commandList) {
+    if ('data' in command && 'execute' in command) {
+      commands.push(command.data.toJSON());
+      client.commands.set(command.data.name, command);
+    } else {
+      console.log(
+        `[WARNING] The command ${command} is missing a required "data" or "execute" property.`
+      );
```

Comment:
This warning made sense when it was dynamic import since it could be importing any file, but now that it's being imported manually, it can be treated as an error since doesn't make sense import files that are not commands in purpose 😅